app/assets/stylesheets/application.scss

```diff
@@ -17,4 +19,9 @@
       padding-right: 0;
     }
   }
+  .call-to-action {
+    background-color: #fff;
```

Comment:
What happens without setting it to white?

Reply:
it goes grey. i think it's the default for cta